### ISSUE TITLE
explicit Resource extendedProps gets nested under extendedProps

### ISSUE THREAD

**Issue by frlinw** [fullcalendar/fullcalendar-scheduler#483]
### Bug Reports

Consistency issue: Event parsing has an `extendedProps` property but Resource Object doesn't.
https://fullcalendar.io/docs/v4/event-parsing
https://fullcalendar.io/docs/v4/resource-parsing

Current Event behavior
```
Raw event object {
  id: '...',
  title: '...',
  extendedProps: {
    customProp: '...'
  }
}
```
```
Event Object {
  id: '...',
  title: '...',
  extendedProps: {
    customProp: '...'
  }
}
```
Current resource behavior
```
Raw resource object {
  id: '...',
  title: '...',
  extendedProps: {
    customProp: '...'
  }
}
```
```
Resource Object {
  id: '...',
  title: '...',
  extendedProps: {
    extendedProps: {
      customProp: '...'
    }
  }
}
```

**Comment by frlinw** [comment]
more relevant issue https://github.com/fullcalendar/fullcalendar-scheduler/issues/482

**Comment by frlinw** [comment]
reopen as the https://github.com/fullcalendar/fullcalendar-scheduler/issues/482 didn't fix it

**Comment by arshaw** [comment]
I'm nearly positive I added this. Would you be able to post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/)?

**Comment by frlinw** [comment]
I tried https://codepen.io/frlinw/pen/vPVREW
but there is an error caused by this file https://rawgit.com/fullcalendar/fullcalendar-scheduler/v4.0.1/dist/resource-timegrid/main.js

Anyway, I can confirm manual `extendedProps` is not processed
![image](https://user-images.githubusercontent.com/11896237/54722317-8b74a100-4b64-11e9-9640-a51233606c67.png)
![image](https://user-images.githubusercontent.com/11896237/54722371-aba46000-4b64-11e9-9e66-a2dd991bb763.png)

**Comment by frlinw** [comment]
@arshaw repro fixed https://codepen.io/frlinw/pen/vPVREW

**Comment by arshaw** [comment]
confirmed, thanks. i see that `extendedProps` is getting nested under `extendedProps`

**Comment by arshaw** [comment]
fixed in v4.1.0
